### PR TITLE
Adds game class properties

### PR DESCRIPTION
**Pull request by DebyGrey**
In this PR I,

- Added the game class properties

### CODE REVIEW

**Comment by luigirazum** [comment]
Hello @DebyGrey, I think it would be better, if you first pull the dev branch, and start your new branch from there.

**Comment by DebyGrey** [comment]
> Hello @DebyGrey, I think it would be better, if you first pull the dev branch, and start your new branch from there.

@luigirazum Alright

File: .DS_Store
Comment (luigirazum) [inline .DS_Store:1]: 👀 **OPTIONAL**
I would like to suggest you avoid adding non-project related files to the repository.
Could you please remove this '.DS_Store' file.

**Review by luigirazum — APPROVED** [review]
STATUS: 🟢 APPROVED!!!  🎉 🎉 🎉
===
Hi @DebyGrey, 🙋‍♂️ I'm Luis your code reviewer. 
---
### Well done !!! 👏 👏 👏 

#### Your project is **complete!**

Now you can merge this branch and move on 🥇 
---
Congratulations! 🎉 🎉 🎉
===
## To Highlight 🎉
- No Linters errors ✔️



## Optional suggestions ✍️
- There is an **OPTIONAL** please follow the 👀 on the commented code.
> _Every comment with the <kbd>OPTIONAL</kbd> prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

***
Cheers and Happy Coding! 👨‍💻
---
Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me `@luigirazum` in your question so I can receive the notification, you can also reach me out on Slack using the same tag as well.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._** 

----
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**